### ISSUE TITLE
not showing in all pages

### ISSUE THREAD

**Issue by Mariaabpr** [t49tran/react-google-recaptcha-v3#26]
I use this package for recaptcha in my react  website. i want to show it in only one page .
this package set in contact page but showing in other pages after open contact page.
help me

**Comment by jpomykala** [comment]
I got the same problem

**Comment by av1at0r** [comment]
I ran into the same problem. I cloned the repository to my computer, built and added to the project. That was enough to solve the problem.

**Comment by spencer741** [comment]
I am having a similar issue. The component is not showing up at all for me. I have verified that I have a registered token with Google. I am also getting **Protected by reCaptcha** in the bottom right hand corner, which means my key and domain is registered with Google. I noticed that the current code is behind the [npm releases](https://www.npmjs.com/package/react-google-recaptcha-v3) by by a couple months; this could explain why @av1at0r was able to get it to work by manually building the source.

**Comment by cristianszwarc** [comment]
**I'd like to hide the reCAPTCHA badge. What is allowed?**
https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed

they tell you their rules and even how to hide it:

```
Note: if you choose to hide the badge, please use
.grecaptcha-badge { visibility: hidden; }
```

**Comment by t49tran** [comment]
That's problem should now be resolved in 1.5.1.
